packages/cli/src/nonInteractiveCli.test.ts

```diff
@@ -341,4 +342,55 @@ describe('runNonInteractive', () => {
     );
     expect(mockProcessExit).not.toHaveBeenCalled();
   });
+
+  it('should log tool usage when showToolUsage is enabled', async () => {
```

Comment:
NIT: For sanity check could you add a negative case so when showToolUsage is false we shouldn't see tool calls in logs